src/org/joml/Rectanglei.java

```diff
@@ -107,26 +107,51 @@ public Rectanglei(int minX, int minY, int maxX, int maxY) {
         this.maxY = maxY;
     }
 
+    /**
+     *
+     * @return width of the rectangle
+     */
     public int width() {
         return maxX - minX;
     }
 
+    /**
+     *
+     * @return height of the rectangle
+     */
     public int height() {
         return maxY - minY;
     }
 
-    public Vector2i size() {
-        return new Vector2i(maxX - minX, maxY - minY);
+    /**
+     * retrieves the size of the rectangle
```

Comment:
Please use a form of documentation that is consistent with the other methods' JavaDocs.
1. Uppercase first letter.
2. Imperative form instead of third-person singular.
3. Ending with a dot as it is a sentence.
4. Consistently use a first sentence/description and not only on `@return` (see other methods as well)